rollup.config.js

```diff
@@ -59,7 +59,20 @@ const commonjsOptions = {
   // https://github.com/rollup/rollup-plugin-commonjs#custom-named-exports
   // https://github.com/reduxjs/react-redux/issues/643#issuecomment-285008041
   namedExports: {
-    react: ['Component', 'createContext']
+    react: [
```

Comment:
This is becoming annoying, especially if someone new tries to use a function from react and don't know about this whitelist.

I don't know exactly why, but it started failing in react-jss package and ssaying those are not exported, demanding to add them

Any ideas @TrySound

Reply:
https://github.com/cssinjs/jss/pull/1426